cypress/e2e/comments.spec.ts

```diff
@@ -175,46 +228,129 @@ describe("Comments", () => {
         // but on `localhost`, cypress is fast enough
         // to execute the commands + assertion
         // before the redirect is done
-        openCommentsDrawer()
-        cy.get(COMMENTS_INPUT_SELECTOR).type(MOCK_COMMENT).blur()
-        submitComment()
```

Comment:
this is safe to remove cos it's rebased off the fix branch right?

Reply:
This is passing test thou, users are supposed to be able to submit comments